### DISCUSSION TITLE
Nornir and Jump Server

### DISCUSSION THREAD

**Discussion by rbot7** [nornir-automation/nornir#778] [Q&A] [answered]
Hi!

I was looking in [Issue 565](https://github.com/ktbyers/netmiko/issues/565) from Netmiko and @ktbyers suggested that Nornir can handle Concurrency better than Netmiko alone. 

I'm not familiar with Nornir, but before dive in I wanted to see if it is possible to use Nornir in a scenary with a jump server that I can only send ssh commands. 
I can't edit any files on the jump server (like sshd_config).

Currently I use Netmiko and write_channels to connect to my devices and after use a redispatch to match the plataform. Also, I'm using concurrent.futures.

Thanks.

**Comment by ktbyers** [comment, marked as answer]
Yes, you can use Nornir and the Nornir-Netmiko plugin to bounce through a Jump Server. So yes you can do this with Nornir.

If you already have a working setup, however, it very likely is not worth doing. In `issue 565` someone was trying to roll their own concurrency and was having issues. I suggested the look at using Nornir instead of trying to implement their own concurrency (which was also my way of saying I was not going to debug their concurrency problems for them).

**Reply by rbot7** [reply]
Thanks for your reply, @ktbyers.
I also thank you for your effort and your contribution to so many projects!
